components/admin/modules/SideBar.test.js

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { render, screen } from '@testing-library/react'
+import { SideBar } from './SideBar'
+
+const moduleData = [
+  {
+    id: 10,
+    name: 'Math',
+    content: 'Cool'
+  },
+  {
+    id: 11,
+    name: 'English',
+    content: 'Bad'
+  },
+  {
+    id: 12,
+    name: 'Math II',
+    content: 'How '
+  }
+]
+
+describe('AdminLessonSideBar component', () => {
+  test('Should render lessons', async () => {
+    render(<SideBar modules={moduleData} selectedModule={1} />)
+    render(<SideBar selectedModule={1} />)
```

Comment:
Why is `render` called twice?

Reply:
One time without data

Reply:
Is it for the test coverage? I feel it could be separated into two tests, one to check if modules are rendered, and one to check the opposite.